core/src/main/java/org/apache/accumulo/core/util/shell/commands/OptUtil.java

```diff
@@ -117,7 +117,7 @@ public static OptionGroup addListDeleteGroup(final String name) {
   }
 
   public static Option startRowOpt() {
-    final Option o = new Option(START_ROW_OPT, "begin-row", true, "begin row (inclusive)");
+    final Option o = new Option(START_ROW_OPT, "begin-row", true, "begin row (NOT) inclusive");
```

Comment:
I can do that.  The reason I used (NOT) inclusive was to make the merge command not change it's current description.  See https://github.com/apache/accumulo/blob/1.6/core/src/main/java/org/apache/accumulo/core/util/shell/commands/MergeCommand.java#L100